webapp-django/crashstats/crashstats/static/crashstats/css/base/layout.less

```diff
@@ -117,3 +117,70 @@ body {
         }
     }
 }
+
+#product-cards {
+    list-style: none;
+    margin: 0 auto;
+    padding: 20px 0 0 0;
+    width: 940px;
+
+    > li {
+        background: #ccc;
+        background: linear-gradient(to bottom, #ddd, #bbb);
+        border-radius: 6px;
+        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+        display: inline-block;
+        margin: 0 8px 20px;
+        min-height: 156px;
+        padding: 30px 1px 1px 1px;
+        position: relative;
+        width: 290px;
+        word-wrap: break-word;
+        z-index: 1;
+
+        text-align: center;
+
+        &:before {
+            background: #fff;
+            border-radius: 5px;
+            bottom: 1px;
+            content: '';
+            left: 1px;
+            position: absolute;
+            right: 1px;
+            top: 1px;
+            z-index: -1;
+        }
+
+        > a {
+            display: block;
+            padding: 20px;
+            text-decoration: none;
+        }
+    }
+
+    a,
+    a:visited {
+        color: @base-link-color;
+        display: block;
+    }
+
+    .productname {
+        font-size: 22px;
+    }
+
+    /* Clearfix */
+    .cf {
+        &:before,
+        &:after {
+            display: table;
+            content: "";
+            // Fixes Opera/contenteditable bug:
+            // http://nicolasgallagher.com/micro-clearfix-hack/#comment-36952
+            line-height: 0;
+        }
+        &:after {
+            clear: both;
+        }
+    }
+}
```

Comment:
This is the CSS I copied and pasted from SUMO and then edited down.

CSS is not my forte. If you have opinions on what to do here, please feel free to fix it.

@Osmose , @lonnen : Thoughts? ^^^